stories/components/OrgAvatar.tsx

```diff
@@ -0,0 +1,88 @@
+import { FC } from "react"
+import styled from "styled-components"
+
+const ImageContainer = styled.div`
+  width: 87px;
+  height: 87px;
+  border: 2.5px solid var(--bs-green); //dependant on the stance
+  border-radius: 50%;
+  position: relative;
+
+  .orgLogo {
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    object-fit: contain;
+    border: 2.5px solid white;
+  }
+
+  .stanceIcon {
+    width: 1.5rem;
+    height: 1.5rem;
+    position: absolute;
+    bottom: -0.75rem;
+    left: calc(50% - 0.75rem);
+  }
+`
+
+const OrgAvatarContainer = styled.div`
+  width: 124px;
+  height: 133px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0px;
+
+  p {
+    font-size: 12px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    margin-top: 0.5rem;
+  }
+`
+
+interface Props {
+  name: string
+  orgImageSrc: string
+  stanceTitle: string
```

Comment:
We call this the [position](https://github.com/robertmrowiec/advocacy-maps/blob/ff7c44fa5a010a0ed2beba95e5e2aef44ff5a9b6/components/db/testimony/types.ts#L19) when dealing with testimony, and I think we can reuse that here. 

```
import { Position } from "components/db"

interface Props {
  position: Position
}